eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -1,18 +1,21 @@
+# cSpell:ignore changedfiles
+# cSpell:ignore Committish
+
 steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+    parameters:
+      DefaultBranchVariableName: DefaultBranch
   - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
     - pwsh: |
         Set-PsDebug -Trace 1
```

Comment:
I don't think this debug tracing is helping much as it is just adding noise. Could you go ahead and remove this as well.